### PR TITLE
New Micro Service Project added with EF migration and employee domain…

### PR DESCRIPTION
**Pull request by atulp789**
… functionality

### CODE REVIEW

File: EmployeeMicroService/WeatherForecast.cs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+namespace EmployeeMicroService;
```
Comment (joseMana) [inline EmployeeMicroService/WeatherForecast.cs:1]: Delet this

Comment (joseMana) [inline EmployeeMicroService/WeatherForecast.cs:1]: delete this

File: EmployeeMicroService/EmployeeMicroService.sln [hunk 2]
```diff
@@ -0,0 +1,22 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
```
Comment (joseMana) [inline EmployeeMicroService/EmployeeMicroService.sln:2]: should be ignored

File: EmployeeMicroService/Data/Migrations/20240805193946_Added new field to cpature.cs [hunk 3]
```diff
@@ -0,0 +1,22 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
```
Comment (joseMana) [inline EmployeeMicroService/Data/Migrations/20240805193946_Added new field to cpature.cs:1]: something wrong with this

File: EmployeeMicroService/Controllers/WeatherForecastController.cs [hunk 4]
```diff
@@ -0,0 +1,31 @@
+
```
Comment (joseMana) [inline EmployeeMicroService/Controllers/WeatherForecastController.cs:1]: delete this

File: EmployeeMicroService/Controllers/EmployeeController.cs [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+    }
+
+    [HttpGet] // api/Get
+    public async Task<ActionResult<IEnumerable<Employee>>> Get()
+    {
+        var list = await _unitOfWork.EmployeeRepository.GetEmployeesAsync();
+        return Ok(list);
+    }
+
+    [HttpGet("{id:int}")] // api/Get/1
```
Comment (joseMana) [inline EmployeeMicroService/Controllers/EmployeeController.cs:43]: any reason for the type "int" in the route?

File: EmployeeMicroService/Data/Migrations/20240805193946_Added new field to cpature.cs (hunk 3 above)
Comment (atulp789) [reply EmployeeMicroService/Data/Migrations/20240805193946_Added new field to cpature.cs:1]: I can see the code it was first migration

**Pull request merged**